### ISSUE TITLE
What is this doing?

### ISSUE THREAD

**Issue by chrisnaubert** [junbuluv/macroproject#3]
https://github.com/junbuluv/macroproject/blob/e5710b8f6f5e505df5da3d9cf4ad84f648e18ff4/PEA/main.m#L63-L66

Per title: What is this doing? It looks like you're trying to find the sigma that makes the residuals the smallest. However, the policies you just solved for are valid only for the value of sigma you solved them with. I.e. if you change sigma, you have to resolve for the policies.

**Comment by junbuluv** [comment]
> https://github.com/junbuluv/macroproject/blob/e5710b8f6f5e505df5da3d9cf4ad84f648e18ff4/PEA/main.m#L63-L66
> 
> Per title: What is this doing? It looks like you're trying to find the sigma that makes the residuals the smallest. However, the policies you just solved for are valid only for the value of sigma you solved them with. I.e. if you change sigma, you have to resolve for the policies.

Yes...that part does not make sense I will either change it so it evaluates new sigma_z, resolving for the policies again or find a new way
